OverlayPlugin.Core/NetworkProcessors/LineCountdown.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using RainbowMage.OverlayPlugin.MemoryProcessors;
+
+namespace RainbowMage.OverlayPlugin.NetworkProcessors
+{
+    [StructLayout(LayoutKind.Explicit, Size = structSize, Pack = 1)]
+    internal unsafe struct Countdown_v655
+    {
+        // 6.5.5 packet data (minus header):
+        // 34120010 4F00 1500 53  00  00  0102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F20 00000000
+        // AAAAAAAA CCCC BBBB DD  EE  FF  GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG HHHHHHHH
+        // 0x0      0x4  0x6  0x8 0x9 0xA 0xB
+        // Actor ID Wrld Time Res Unk Unk Name
+
+        public const int structSize = 48;
+        [FieldOffset(0x0)]
+        public uint actorID;
+        [FieldOffset(0x4)]
+        public ushort senderWorldId;
+
+        [FieldOffset(0x6)]
+        public ushort time;
```

Comment:
This is admittedly a bit ticky-tacky, but I'm wondering whether we could call this something a little more explicit, like `countdownTimer`?  (I know the ushort name isn't exposed, but it did take me a second (no pun intended) to realize what `time` meant in this context, since we also write the timestamp in the log.

Reply:
I have renamed all the props to hopefully make more sense at a glance.